core/src/main/java/com/alibaba/alink/params/dataproc/StratifiedSampleWithSizeParams.java

```diff
@@ -0,0 +1,99 @@
+package com.alibaba.alink.params.dataproc;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.flink.ml.api.misc.param.ParamInfo;
+import org.apache.flink.ml.api.misc.param.ParamInfoFactory;
+import org.apache.flink.ml.api.misc.param.ParamValidator;
+import org.apache.flink.ml.api.misc.param.WithParams;
+
+public interface StratifiedSampleWithSizeParams<T> extends
+        WithParams<T> {
+
+
+    ParamInfo<String> GROUP_KEY = ParamInfoFactory
+            .createParamInfo("groupKey", String.class)
+            .setDescription("name of group key")
+            .setRequired()
+            .build();
+
+    ParamInfo<String> SIZE = ParamInfoFactory
+            .createParamInfo("size", String.class)
+            .setDescription("sampling ratio, it should be int value when as a number , defintion format as name1:number1,name2:number2 when as a string")
+            .setRequired()
+            .setValidator(new RatioValidator())
+            .build();
+
+    ParamInfo<Long> SEED = ParamInfoFactory
+            .createParamInfo("seed", Long.class)
+            .setDescription("seed of random")
+            .setOptional()
+            .build();
+
+    ParamInfo<Boolean> WITH_REPLACEMENT = ParamInfoFactory
```

Comment:
Params can be reused.

Reply:
thanks for code review  and suggestions, I had refactoring it